src/main/ts/components/Editor.tsx

```diff
@@ -210,16 +200,20 @@ export class Editor extends React.Component<IAllProps> {
       toolbar: this.props.toolbar || (this.props.init && this.props.init.toolbar),
       setup: (editor) => {
         this.editor = editor;
-        editor.on('init', this.handleInit);
+        this.bindHandlers({});
 
         if (this.props.init && isFunction(this.props.init.setup)) {
           this.props.init.setup(editor);
         }
       }
     };
-
-    if (isTextarea(this.elementRef.current)) {
-      this.elementRef.current.style.visibility = '';
+    if (!this.inline) {
```

Comment:
Is this a bugfix separate to the INT-2325 fix? If so, maybe add this to the changelog? And add a changelog entry for INT-2325 itself?

Reply:
Well the lines after this were to ensure we had the initialValue as the actual initial value. Because I changed the `isTextarea` function so I could match the code in tinymce I had to modify this because this should only happen for non-inline elements as far as I can tell.

But yes I need a changelog entry for INT-2325.

Reply:
Added changelog entry.